lib/ex_webrtc/peer_connection.ex

```diff
@@ -377,11 +418,16 @@ defmodule ExWebRTC.PeerConnection do
   end
 
   @impl true
-  def handle_info({:dtls_transport, _pid, {:rtp_data, data}}, state) do
-    case Demuxer.demux(state.demuxer, data) do
-      {:ok, demuxer, mid, packet} ->
-        notify(state.owner, {:data, {mid, packet}})
-        {:noreply, %__MODULE__{state | demuxer: demuxer}}
+  def handle_info({:dtls_transport, _pid, {:rtp, data}}, state) do
+    with {:ok, demuxer, mid, packet} <- Demuxer.demux(state.demuxer, data),
+         %RTPTransceiver{} = t <- Enum.find(state.transceivers, &(&1.mid == mid)) do
```

Comment:
Not sure about this one. I belive this should be in the `do end` body as this should always be true

Reply:
Yeah, but atm demuxer does not check that, I could change this in this pull request, but the whole receiving/sending data business is quite temporary anyways, isn't it?